### PR TITLE
CMIP guide/CMIP6 license updates

### LINKED ISSUE #464: CMIP guide/CMIP6 license updates
The following links are broken on both [Accessing model output](https://pcmdi.llnl.gov/CMIP6/Guide/dataUsers.html#3-accessing-model-output) and [Model output requirements](https://pcmdi.llnl.gov/CMIP6/Guide/modelers.html#5-model-output-requirements):

- [Information on pressure levels](https://www.earthsystemcog.org/site_media/projects/wip/CMIP6_pressure_levels.pdf?id=86)
- [Guidance on time-averaging](https://www.earthsystemcog.org/projects/wip/time_averages)

The first link should be to: https://cmip6dr.github.io/Data_Request_Home/Documents/CMIP6_pressure_levels.pdf 

I don't know where the second document is.

### PR DESCRIPTION
**Pull request by durack1**
Fix #464 
Fix WCRP-CMIP/WGCM_Infrastructure_Panel_priv#4
Ref WCRP-CMIP/CMIP6_CVs#1062

@matthew-mizielinksi @taylor13 @CMIP-IPO here is a first pass at updates now we have the licenses relaxed across the archive

@mauzey1 we'll need to ensure that we have the [updated CMIP6 licenses page](https://wcrp-cmip.github.io/CMIP6_CVs/docs/CMIP6_source_id_licenses.html) linked off the https://pcmdi.llnl.gov/CMIP6/ArchiveStatistics/esgf_data_holdings/ subpages

@MartinaSt is there anything left to do with regard to WCRP-CMIP/CMIP6_CVs#1062?

@matthew-mizielinski how about the WIP pages? e.g. WCRP-CMIP/WGCM_Infrastructure_Panel#1, WCRP-CMIP/WGCM_Infrastructure_Panel#7, WCRP-CMIP/WGCM_Infrastructure_Panel#29?

### CODE REVIEW

**Comment by MartinaSt** [comment]
@durack1 As the licenses are stable now, yes, we could start to work on https://github.com/WCRP-CMIP/CMIP6_CVs/issues/1062 . If you need additional information from me, please comment there.

File: CMIP6/TermsOfUse/TermsOfUse6-2.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
+layout: default
+title:  CMIP6 Terms of Use
+---
+
+# CMIP6: Terms of Use
+
+To enable modeling groups and others who support CMIP6 to demonstrate its impact (and secure ongoing funding), you are required to cite and acknowledge those who have made CMIP6 possible. You also must abide by any licensing restrictions. Specifically, you must
+
+* Adhere to licensing restrictions associated with the Creative Commons Attribution 4.0 International ([CC BY 4.0]) license. *Note as of October 2022 all CMIP6 contributing modeling groups have relaxed the license associated with their climate model output data. The license information available from the [CMIP6 Climate Model Data license page] supercedes the information sstated in the “license” global attribute found in the netCDF file headers of each CMIP6 output file.
```
Comment (durack1) [inline CMIP6/TermsOfUse/TermsOfUse6-2.md:10]: Correct "sstated" to "stated"

**Comment by durack1** [comment]
@matthew-mizielinski ping - I will tag you as a reviewer when you have accepted the invite

**Comment by mauzey1** [comment]
> @mauzey1 we'll need to ensure that we have the [updated CMIP6 licenses page](https://wcrp-cmip.github.io/CMIP6_CVs/docs/CMIP6_source_id_licenses.html) linked off the https://pcmdi.llnl.gov/CMIP6/ArchiveStatistics/esgf_data_holdings/ subpages

@durack1 So do we want each page in ESGF data holdings to have a link to the [CMIP6 licenses page](https://wcrp-cmip.github.io/CMIP6_CVs/docs/CMIP6_source_id_licenses.html) somewhere at the top of the page?

**Comment by mauzey1** [comment]
Screenshot 2022-10-18 at 14-40-11 ESGF CMIP6 Data Holdings

@durack1 This is currently what the top of the pages look like.  Where do you think we should squeeze in those 4 links?

**Comment by durack1** [comment]
@mauzey1 could we use the right hand side whitespace to put in these links - maybe in a grid? I'm happy to follow your lead on this

File: CMIP6/Guide/dataUsers.md [hunk 2]
```diff
@@ -130,12 +130,12 @@ Information is under preparation describing the governance of the following:
...
 [experimentIdhtml]: https://wcrp-cmip.github.io/CMIP6_CVs/docs/CMIP6_experiment_id.html
-[cmip6DataRequestCog]: https://www.earthsystemcog.org/projects/wip/CMIP6DataRequest
+[cmip6DataRequest]: https://cmip6dr.github.io/Data_Request_Home
 [cfConventionsPage]: http://cfconventions.org/
 [cmip6GlobalAttGoogleDoc]: http://goo.gl/v1drZl
 [cmip6GridGoogleDoc]: http://goo.gl/1oA7bO
-[cmip6PressureLevelsPdf]: https://www.earthsystemcog.org/site_media/projects/wip/CMIP6_pressure_levels.pdf
-[cmip6TimeAveragesCog]: https://www.earthsystemcog.org/projects/wip/time_averages
+[cmip6PressureLevelsPdf]: https://cmip6dr.github.io/Data_Request_Home/Documents/CMIP6_pressure_levels.pdf
+[cmip6TimeAveragesCog]: https://web.archive.org/web/20201020095856/https://www.earthsystemcog.org/projects/wip/time_averages
```
Comment (matthew-mizielinski) [inline CMIP6/Guide/dataUsers.md:138]: Ported to WIP pages: https://wcrp-cmip.github.io/WGCM_Infrastructure_Panel/CMIP6/time_and_area_averaging.html

File: CMIP6/Guide/modelers.md [hunk 3]
```diff
@@ -137,13 +137,13 @@ be available soon. Any changes made to a released dataset will be documented in
...
+The [CMIP6 Data Request][cmip6DataRequest] defines the variables that should
 be archived for each experiment and specifies the time intervals for which they
 should be reported. It provides much of the variable-specific metadata that should
 be stored along with the data. It also provides tools for estimating the data storage
 requirements for CMIP6.
 
-Additional information about the data request is available at https://www.earthsystemcog.org/projects/wip
+Additional information about the data request is available at https://cmip6dr.github.io/Data_Request_Home
 
 *[Further explanation will be added here.]*
```
Comment (matthew-mizielinski) [inline CMIP6/Guide/modelers.md:148]: remove?

File: CMIP6/Guide/modelers.md [hunk 4]
```diff
@@ -411,12 +411,12 @@ Information is under preparation describing the governance of the following:
...
 [input4mipsGoogleDoc]: http://goo.gl/r8up31
-[cmip6DataRequestCog]: https://www.earthsystemcog.org/projects/wip/CMIP6DataRequest
+[cmip6DataRequest]: https://cmip6dr.github.io/Data_Request_Home
 [cfConventionsPage]: http://cfconventions.org/
 [cmip6GlobalAttGoogleDoc]: http://goo.gl/v1drZl
 [cmip6GridGoogleDoc]: http://goo.gl/1oA7bO
-[cmip6PressureLevelsPdf]: https://www.earthsystemcog.org/site_media/projects/wip/CMIP6_pressure_levels.pdf
-[cmip6TimeAveragesCog]: https://www.earthsystemcog.org/projects/wip/time_averages
+[cmip6PressureLevelsPdf]: https://cmip6dr.github.io/Data_Request_Home/Documents/CMIP6_pressure_levels.pdf
+[cmip6TimeAveragesCog]: https://web.archive.org/web/20201020095856/https://www.earthsystemcog.org/projects/wip/time_averages
```
Comment (matthew-mizielinski) [inline CMIP6/Guide/modelers.md:419]: see comment above for page on WIP website

**Review by matthew-mizielinski — APPROVED** [review]
Some minor items, but it looks good to me from my quick run through. I've put in what I think is the fix required for
https://github.com/WCRP-CMIP/WGCM_Infrastructure_Panel/issues/7

**Comment by durack1** [comment]
@mauzey1 can you please review and merge - if no issues?

**Comment by mauzey1** [comment]
@durack1  Would something like this be sufficient?

Screenshot 2022-10-19 at 17-18-04 ESGF CMIP6 Data Holdings

**Comment by durack1** [comment]
@mauzey1 this is a good start, can I suggest we standardize the nomenclature so:

WCRP-CMIP/CMIP6_CVs info: experiments/experiment_id  |  institutions/institution_id  |  models/source_id  |  model data licenses

And if these can open in a new tab, means that we're not breaking the existing nav

**Comment by mauzey1** [comment]
@durack1  Like this?  The links will open a new tab when clicked.

Screenshot 2022-10-19 at 17-53-00 ESGF CMIP6 CMIP Data Holdings

**Comment by durack1** [comment]
@mauzey1 looks good to me! thanks

**Review by mauzey1 — APPROVED** [review]

**Pull request merged**